lib/bundler/audit/scanner.rb

```diff
@@ -41,10 +41,13 @@ class Scanner
       #
       def initialize(root=Dir.pwd,gemfile_lock='Gemfile.lock')
         @root     = File.expand_path(root)
+        gemfile_lock_path = File.join(@root,gemfile_lock)
+        unless File.file?(gem_file_lock_path)
+          puts "Unable to find the gem's Gemfile.lock file in #{@root}!"
```

Comment:
Never put CLI printing messages or call `exit` in regular library code. This should raise a custom exception (ex: `GemfileLockNotFound`), which is then caught in `Bundler::Audit::CLI` and an error message printed there.